src/api/app/jobs/populate_to_sphinx_job.rb

```diff
@@ -0,0 +1,20 @@
+class PopulateToSphinxJob < ApplicationJob
+  ALLOWED_MODELS = {
+    package: Package,
+    project: Project
+  }.freeze
+
+  queue_as :quick
+
+  def perform(id:, model_name:)
+    model_class = ALLOWED_MODELS[model_name]
```

Comment:
`ALLOWED_MODELS.fetch(model_name)` is better because the exception handling is done automatically when a lookup fails on fetch since it raises a KeyError

Reply:
Sure!